akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md

```diff
@@ -183,6 +183,11 @@ Classic remoting is deprecated but can be used in `2.6.` Explicitly disable Arte
 specific to classic remoting needs to be moved to `akka.remote.classic`. To see which configuration options
 are specific to classic search for them in: [`akka-remote/reference.conf`](/akka-remote/src/main/resources/reference.conf)
 
+## Java Serialization
+
+Consistent hashing is serializing the hash key, unless it's bytes or String. If you use consistent hashing router
```

Comment:
First sentence reads a bit odd, maybe start with the problem / user feature affected e.g.

`When using a consistent hashing router keys that were not bytes or a String were previously using the Java serializer. `

Reply:
Well, it wasn't using Java serialization specifically. It used whatever serializer that was bound, and that could have been JavaSerializer. Improved wording.